Svc/PolyDb/test/ut/PolyDbTesterHelpers.cpp

```diff
@@ -0,0 +1,38 @@
+// ======================================================================
```

Comment:
You could take this opportunity to switch to the auto-helpers setup which autocodes this file.